api/maven-api-meta/pom.xml

```diff
@@ -27,6 +27,7 @@
   </parent>
 
   <artifactId>maven-api-meta</artifactId>
-  <name>Maven API Meta annotations</name>
+  <name>Maven 4 API Meta annotations</name>
```

Comment:
Annotations

Reply:
should we rename the artifactId?

Reply:
Yes, good idea: `maven-api-meta-annotations`?